ui/apps/platform/src/global/initializeAnalytics.js

```diff
@@ -7,7 +7,7 @@
 // the code below is generated from segment api with the exception of the writeKey/userId parameters as well as the analyticsIdentity call
 // segment intends for the write key to be hardcoded but we are pulling it from the telemetry config service call and adding it using a parameter
 
-export function initializeAnalytics(writeKey, userId) {
+export function initializeAnalytics(writeKey, proxyApiEndpoint, userId) {
```

Comment:
nit: I saw you happened to destructure the properties in this order before you called this function, but I like to make a habit of always _adding_ params at the end of a functions argument list, so that it can be backwards compatible. In this case, it doesn't matter, as you control the only call sites tightly. It's just a good habit to get into.

```suggestion
export function initializeAnalytics(writeKey, userId, proxyApiEndpoint) {
```